src/Nethermind/Nethermind.WebSockets/SocketClientBase.cs

```diff
@@ -0,0 +1,114 @@
+using System;
+using System.Buffers;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.WebSockets;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Nethermind.Logging;
+using Nethermind.Serialization.Json;
+
+namespace Nethermind.WebSockets
+{
+    public class SocketClientBase : ISocketsClient
```

Comment:
Either abstract or drop "Base" suffix?

Reply:
We have instances of this class so it cannot be abstract.
The name change was suggested by Tomasz in his review.